.github/workflows/platformDeploy.yml

```
@@ -292,6 +292,7 @@ jobs:
         with:
           PR_LIST: ${{ steps.getReleasePRList.outputs.PR_LIST }}
           IS_PRODUCTION_DEPLOY: ${{ env.SHOULD_DEPLOY_PRODUCTION == 'true' }}
+          VERSION: ${{ env.version }}
```

```suggestion
          VERSION: ${{ env.VERSION }}
```

Not sure if it's case sensitive, but just to be sure.